libs/components/src/lib/card/live-market/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { qtMerge, Text, Button } from '@deriv/quill-design';
+import {
+  StandaloneArrowUpRegularIcon,
+  StandaloneArrowDownRegularIcon,
+  StandaloneArrowRightRegularIcon,
+} from '@deriv/quill-icons';
+import { LiveMarketContent } from '../types';
+import LivePrice from './live-price';
+import clsx from 'clsx';
+
+export interface LiveMarketCardProps extends LiveMarketContent {
+  className?: string;
+}
+
+export const LiveMarketCard: React.FC<LiveMarketCardProps> = ({
+  className,
+  instrumentIcon,
+  instrument,
+  changePercentage,
+  status,
+  bidPrice,
+  askPrice,
+  spread,
+}) => {
+  const colorVariant = {
+    up: '#00ae7a',
+    down: '#de0040',
+    remain: '#0000007a',
+    closed: '#0000003d',
+  };
+
+  return (
+    <div
+      className={qtMerge(
+        'flex flex-col',
+        'rounded-pill',
+        'h-[262px] w-[286px]',
+        'bg-background-primary-container',
+        'p-general-lg lg:p-general-xl',
+        'gap-gap-xl',
+        className,
+        status === 'closed' ? 'opacity-700' : '',
+      )}
+    >
+      <div className="flex h-[52px] flex-row justify-between">
+        <div className="flex flex-row gap-gap-md">
+          {instrumentIcon}
+          <Text size="md" className="pt-general-xs">
+            {instrument}
+          </Text>
+        </div>
+        <div className="flex flex-row gap-gap-md pt-[2px]">
+          {status === 'up' ? (
+            <StandaloneArrowUpRegularIcon
+              iconSize="sm"
+              fill={colorVariant[status]}
+            />
+          ) : status === 'down' ? (
+            <StandaloneArrowDownRegularIcon
+              iconSize="sm"
+              fill={colorVariant[status]}
+            />
+          ) : (
+            <StandaloneArrowRightRegularIcon
+              iconSize="sm"
+              fill={colorVariant[status]}
+            />
+          )}
```

Comment:
What is the difference here? No need to add a condition if we are displaying thesame for all of them. 
```suggestion
       
            <StandaloneArrowRightRegularIcon
              iconSize="sm"
              fill={colorVariant[status]}
            />
         
```

Reply:
Different icon based on the status. currently I have just used the sample icons. will update once the charticons are ready in quill-icons.

Reply:
Those icons can be added as an object 

```
 const icons = {
  up: <StandaloneArrowDownRegularIcon
              iconSize="sm"
              fill={colorVariant[status]}
            />,
    down: <StandaloneArrowDownRegularIcon
              iconSize="sm"
              fill={colorVariant[status]}
            />
            }